src/components/PopoverProvider/index.js

```diff
@@ -22,7 +22,11 @@ function PopoverContextProvider(props) {
         if (!activePopoverRef.current || (anchorRef && anchorRef !== activePopoverRef.current.anchorRef)) {
             return;
         }
+
         activePopoverRef.current.close();
+        if (activePopoverRef && activePopoverRef.current && activePopoverRef.current.onCloseCallback) {
```

Comment:
I don't think we need `activePopoverRef && activePopoverRef.current &&` since above we're calling `activePopoverRef.current.close();`
```suggestion
        if (activePopoverRef.current.onCloseCallback) {
```